src/index.css

```
@@ -2,557 +2,8 @@
 
 @custom-variant tablet (@media (max-width: 1280px));
 
-* {
-  font-family: "Pretendard";
-}
+@import "./styles/tokens.css";
```

Stylelint (17.11.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)